index.html

```diff
@@ -270,7 +270,7 @@ <h3>
               <li>VC JSON Web Tokens</li>
             </ul>
             </li>
-            <li>Extensions for binding multilingual strings to <a href="https://www.gnu.org/software/gettext/" title="GNU gettext">GNU gettext</a>-like localization files</li>
+            <li>Extensions for binding multilingual resources for localized user interfaces.</li>
```

Comment:
The change I was proposing was for this string...

Reply:
Ah, right... my bad.

Reply:
This part of the text (in 2.2 Other Deliverables) discusses a non-normative document on external mapping objects which we develop. So I think this should be ok.

I thought your request is for Section 1. Scope.

Reply:
@Sakurann could you suggest text which you would find acceptable here?